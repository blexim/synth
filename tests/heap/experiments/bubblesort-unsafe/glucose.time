WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 10
0.41user 0.01system 0:00.42elapsed 100%CPU (0avgtext+0avgdata 29852maxresident)k
0inputs+16outputs (0major+10009minor)pagefaults 0swaps
