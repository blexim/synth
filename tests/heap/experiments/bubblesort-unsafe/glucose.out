c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:         42692                                                                   |
c |  Number of clauses:          159439                                                                   |
c |  Parse time:                   0.05 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           1.59 Mb                                                                |
c |  Simplification time:          0.18 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c last restart ## conflicts  :  44 140 
c =========================================================================================================
c restarts              : 29 (192 conflicts in avg)
c blocked restarts      : 0 (multiple: 0) 
c last block at restart : 0
c nb ReduceDB           : 1
c nb removed Clauses    : 985
c nb learnts DL2        : 324
c nb learnts size 2     : 114
c nb learnts size 1     : 15
c conflicts             : 5573           (13559 /sec)
c decisions             : 14608          (0.00 % random) (35541 /sec)
c propagations          : 1297549        (3156877 /sec)
c conflict literals     : 73338          (25.89 % deleted)
c nb reduced Clauses    : 84
c CPU time              : 0.411023 s

s SATISFIABLE
